src/openzaak/components/besluiten/models.py

```diff
@@ -177,7 +210,8 @@ def __init__(self, *args, **kwargs):
         # self._previous_zaak = self.zaak
 
         self._previous_zaak = self._zaak
-        self._previous_zaak_url = self._zaak_url
+        self._previous_zaak_base_url_id = self._zaak_base_url_id
+        self._previous_zaak_relative_url = self._zaak_relative_url
```

Comment:
can't you use the model field machinery to just store the previous URL reference? I'd expect `self._zaak_url` here to still return the fully qualified URL. Then you would not have to make any changes in `def previous_zaak` and the `urljoin` machinery is abstracted away in the model field.

Reply:
you are right, the change is excessive. I'll revoke this change